force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -122,13 +199,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of all integers up to the limit.
      */
     public static Integer sumUpToLimit(Integer intLimit) {
-        /* 
-         for (){
-            
+        Integer sumOfAll = 0; 
+        Integer i;
+        for (i = 1; i <= intLimit; i++){
+            sumOfAll = sumOfAll + i;
```

Comment:
You can use the addition assignment operator here: https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_expressions_operators_understanding.htm#:~:text=Addition%20assignment%20operator